crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -133,10 +135,20 @@ impl Consensus {
                 return Err(BlockValidationErrors::FirstTxIsnNotCoinbase.into());
             }
             // Amount of all outputs
-            let output_value = transaction
-                .output
-                .iter()
-                .fold(0, |acc, tx| acc + tx.value.to_sat());
+            let output_value = transaction.output.iter().fold(0, |acc, tx| {
+                acc + {
+                    if tx.value.to_sat() > 0 {
```

Comment:
Can this value be negative? If it is, we should err, not log

Reply:
That's an old approach to validate that any value is negative.
Now it's properly returning a error.
Using this approach i could not return a error, only log that im aborting and invalidating the whole block